README.rst

```diff
@@ -116,6 +119,52 @@ Usage
       latest     Get the latest tag in a changelog.
       tag        Create an annotated git tag based on a changelog entry.
       update     Updates the version and date of the most recent changelog entry.
+      create     Creates a new CHANGELOG.md file with a standard header.
+      entry      Add a quick entry to the current version.
+      next       Create a new unreleased version in changelog file.
+
+next
+----
+
+Create a new unreleased version in changelog file.
+
+::
+
+   Usage: chag next
+
+   Creates a new unreleased version inside of the changelog file.
+
+   Options:
+     --help     Displays this message.
+
+entry
```

Comment:
I should also note that highjacking other people's PRs and then trying to "supersede" them with your own is generally poor open source etiquette. If you would like to build on my PR, one way of doing so is by rebasing your PR against mine if mine it is merged, or if you want to implement the same features but differently then you can do so, but not including my commits on your own and then commenting on my PR as if I agreed with the changes you added to my changes.

Thanks for your own source interest.